index.md

```diff
@@ -28,13 +28,13 @@ customers happy.
     <span class="card-heading">For Developers</span>
     <p>Use these resources to learn and implement Helix patterns.</p>
     <ul>
-      <li><a href="#">Pattern Guidelines</a></li>
-      <li><a href="#">UI Framework</a></li>
-      <li><a href="#">Component Explorer</a></li>
+      <li><a href="http://helix.rax.io/">Pattern Guidelines</a></li>
```

Comment:
I wonder if we remove this link for the time being. Seems like all the links in this card should go to the helix-ui side, but there isn't really anything that related to pattern guidelines at the moment. 

Thoughts @araiford